flowzone.yml

```diff
@@ -262,12 +262,26 @@
       fi
 
   - &jsonArrayBuilder
-    name: Build JSON array from comma-separated list
+    name: Build JSON array
     id: to_json_array
-    uses: kanga333/json-array-builder@c7cd9d3a8b17cd368e9c2210bc3c16b0e2714ce5 # v0.2.1
-    with:
-      cmd: bash -c "echo $INPUT | tr -d '[:space:]'"
-      separator: ","
+    shell: bash
+    run: |
+      set -x
+
+      # Remove spaces and newlines from the input
+      INPUT="$(echo "${INPUT}" | tr -d '[:space:]')"
```

Comment:
I think this is deleting all spaces, not just those around the quoted elements, but it's also not new to this PR